test/linuxRules.cpp

```diff
@@ -777,7 +812,11 @@ TEST( Events, DispatchEvent )
     eventSize = sizeof(SYSMON_EVENT_HEADER);
 
     ptr = (char*)(pc + 1);
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wformat-truncation"
+#pragma GCC diagnostic ignored "-Wstringop-overflow"
     strcpy(ptr, "/foo/bar/image");
```

Comment:
Do we know the buffer size here, and can we use `strncpy` rather than disabling the warnings?